backend/compact-connect/stacks/api_stack/v1_api/staff_users.py

```diff
@@ -0,0 +1,662 @@
+                ],
+                additional_properties=False,
+                properties=self._common_user_properties
+            )
+        )
+        return self.api.v1_post_user_request_model
+
+    @property
+    def _attributes_schema(self):
+        return JsonSchema(
+            type=JsonSchemaType.OBJECT,
+            additional_properties=False,
+            properties={
+                'email': JsonSchema(type=JsonSchemaType.STRING, min_length=5, max_length=100),
+                'givenName': JsonSchema(type=JsonSchemaType.STRING, min_length=1, max_length=100),
+                'familyName': JsonSchema(type=JsonSchemaType.STRING, min_length=1, max_length=100)
+            }
+        )
+
+    @property
+    def _patch_attributes_schema(self):
+        """
+        No support for changing a user's email address.
+        """
+        return JsonSchema(
+            type=JsonSchemaType.OBJECT,
+            additional_properties=False,
+            properties={
+                'givenName': JsonSchema(type=JsonSchemaType.STRING, min_length=1, max_length=100),
+                'familyName': JsonSchema(type=JsonSchemaType.STRING, min_length=1, max_length=100)
+            }
+        )
+
+    @property
+    def _permissions_schema(self):
+        return JsonSchema(
+            type=JsonSchemaType.OBJECT,
+            additional_properties=JsonSchema(
+                type=JsonSchemaType.OBJECT,
+                additional_properties=False,
+                properties={
+                    'actions': JsonSchema(
+                        type=JsonSchemaType.OBJECT,
+                        properties={
+                            'read': JsonSchema(type=JsonSchemaType.BOOLEAN),
+                            'admin': JsonSchema(type=JsonSchemaType.BOOLEAN)
+                        }
+                    ),
+                    'jurisdictions': JsonSchema(
+                        type=JsonSchemaType.OBJECT,
+                        additional_properties=JsonSchema(
+                            type=JsonSchemaType.OBJECT,
+                            properties={
+                                'actions': JsonSchema(
+                                    type=JsonSchemaType.OBJECT,
+                                    additional_properties=False,
+                                    properties={
+                                        'write': JsonSchema(type=JsonSchemaType.BOOLEAN),
+                                        'admin': JsonSchema(type=JsonSchemaType.BOOLEAN)
+                                    }
+                                )
+                            }
+                        )
+                    )
+                }
+            )
+        )
+
+    @property
+    def _common_user_properties(self):
+        return {
+            'attributes': self._attributes_schema,
+            'permissions': self._permissions_schema
+        }
+
+    @property
+    def _user_response_schema(self):
```

Comment:
If I understood correctly, I think @landonshumway-ia did some consolidation of these schemas in #218. Is there a plan to merge?

Reply:
Yes. I would like to update this to adopt @landonshumway-ia's newer pattern but didn't do that for this PR, since I was working this in parallel with him establishing the pattern. Same goes for his new CDK tests.

Reply:
got it.